src/Features/SupportQueryString/BrowserTest.php

```diff
@@ -555,6 +555,169 @@ public function render() { return <<<'HTML'
             ->assertQueryStringMissing('filter')
         ;
     }
+
+    /** @test */
+    public function can_handle_empty_querystring_value_as_empty_string()
```

Comment:
I think any of these tests that only care about how values are treated on initial load can be put inside the UnitTest file so that they run much faster and can be used better to refactor the implementation.

Reply:
I added a couple of unit tests